src/ONYXKEYS.ts

```diff
@@ -142,11 +142,11 @@ const ONYXKEYS = {
     /** The user's bank accounts */
     BANK_ACCOUNT_LIST: 'bankAccountList',
 
-    /** The user's credit cards (renamed from cardList) */
+    /** The user's cash card and imported cards */
```

Comment:
fundList: The user's payment and P2P cards

cardList: The user's cash card and imported cards (including the Expensify Card)